core/src/test/scala/integration/kafka/api/IntegrationTestHarness.scala

```diff
@@ -131,13 +137,26 @@ abstract class IntegrationTestHarness extends KafkaServerTestHarness {
     consumer
   }
 
+  def createAdminClient(configOverrides: Properties = new Properties): AdminClient = {
+    val props = new Properties
+    props ++= adminClientConfig
+    props ++= configOverrides
+    val adminClient = AdminClient.create(props)
+    adminClients += adminClient
```

Comment:
It's very easy to overlook `adminClients` vs `adminClient`. Could we name `adminClients` as `adminClientList` or `adminClientCollection`?

Reply:
I slightly prefer the more concise name. Since they have different types, I think it's not likely to cause too much confusion. Since `consumers` and `producers` are named similarly, maybe we could address this in a separate PR.

Reply:
Yeah, let's keep it concise please. I have never seen a bug due to this.